### PR TITLE
add moment for BART distribution

### PR DESCRIPTION
**Pull request by aloctavodia**
Following discussion in #5078

### CODE REVIEW

File: pymc/bart/bart.py [hunk 1]
```diff
@@ -132,6 +137,13 @@ def logp(x, *inputs):
         """
         return at.zeros_like(x)
 
+    @classmethod
+    def get_moment(cls, rv, size, *rv_inputs):
```
Comment (ricardoV94) [inline pymc/bart/bart.py:140]: Could BART shape be determined by the shape of the rv_inputs? Or does the mean already integrate this info?

File: pymc/bart/bart.py [hunk 2]
```diff
@@ -132,6 +137,13 @@ def logp(x, *inputs):
...
+        mean = rv.Y.mean()
+        if not rv_size_is_none(size):
```
Comment (ricardoV94) [inline pymc/bart/bart.py]: Can you add a condition that hits this branch?

File: pymc/bart/bart.py (hunk 1 above)
Comment (aloctavodia) [reply pymc/bart/bart.py:140]: The shape of BART is always the shape of observed data Y.

Comment (ricardoV94) [reply pymc/bart/bart.py:140]: And do you do anything with the size variable or is that branch below unnecessary?

Comment (aloctavodia) [reply pymc/bart/bart.py:140]: everything is a lie :-) Initially I tough about having a size argument for future extension. But probably that will not be very useful.

**Review by ricardoV94 — APPROVED** [review]

**Pull request merged**